source/classes.tex

```diff
@@ -6509,16 +6509,16 @@
 is usable\iref{class.compare.default}.
 
 \pnum
-The return value \tcode{V} of a defaulted \tcode{==} operator function
+The return value of a defaulted \tcode{==} operator function
 with parameters \tcode{x} and \tcode{y} is determined
 by comparing corresponding elements $\tcode{x}_i$ and $\tcode{y}_i$
 in the expanded lists of subobjects for \tcode{x} and \tcode{y}
 (in increasing index order)
 until the first index $i$
 where $\tcode{x}_i\tcode{ == }\tcode{y}_i$ yields a result value which,
 when contextually converted to \tcode{bool}, yields \tcode{false}.
-If no such index exists, \tcode{V} is \tcode{true}.
-Otherwise, \tcode{V} is \tcode{false}.
+If no such index exists, the return value is \tcode{true}.
```

Comment:
Could we say "The return value is `true` if no such index exists, and `false` otherwise."? Then we wouldn't have to repeat "return value".